ebs_snapshots/snapshot_manager.py

```diff
@@ -49,7 +49,7 @@ def _create_snapshot(connection, volume, name=''):
     if not name:
         name = '{}-snapshot'.format(volume.id)
     connection.create_tags(
-        [snapshot.id], dict(Name=name, Creator='ebs-snapshots'))
+        [snapshot.id], dict(Name=name, creator='ebs-snapshots'))
```

Comment:
Though maybe I should add things like `application`, `env` and `version` to this. maybe that is a separate PR?